pages/maintenance.html

```diff
@@ -22,15 +22,15 @@ <h2>Downloads</h2>
 
 	<h2>Community</h2>
 	<ul>
-		<li><a href="https://ask.godotengine.org/">Questions &amp; Answers</a></li>
+		<li><a href="https://forum.godotengine.org/">Forum</a></li>
 		<li><a href="https://discord.gg/4JBkykG">Discord</a></li>
 		<li><a href="https://web.libera.chat/#godotengine">IRC</a></li>
 		<li><a href="https://matrix.to/#/#godotengine:matrix.org">Matrix</a></li>
 		<li><a href="https://www.facebook.com/groups/godotengine/">Facebook</a></li>
 		<li><a href="https://reddit.com/r/godot/">Reddit</a></li>
 		<li><a href="https://github.com/godotengine/godot">GitHub</a></li>
 		<li><a href="https://twitter.com/godotengine">Twitter</a></li>
-		<li><a href="https://godotforums.org/">Forum</a></li>
+		<li><a href="https://godotforums.org/">Unofficial Forum</a></li>
```

Comment:
```suggestion
		<li><a href="https://godotforums.org/">Community Forum</a></li>
```
I would edit this to be Community Forum to be consistent with the description of it from our description at https://godotengine.org/community/